packages/language-server/src/prisma-fmt/nativeTypes.ts

```diff
@@ -11,13 +11,13 @@ export default function nativeTypeConstructors(
   text: string,
   onError?: (errorMessage: string) => void,
 ): NativeTypeConstructors[] {
-  console.log("running native_types() from prisma-fmt");
+  console.log('running native_types() from prisma-fmt')
   try {
     const result = prismaFmt.native_types(text)
     return JSON.parse(result) as NativeTypeConstructors[]
   } catch (err) {
     if (onError) {
-      onError(`${err}`)
+      onError(`prisma-fmt error'd during getting available native types. ${err}`)
```

Comment:
Semi-OT: Raises the question how we could test this. Probably best manually, via making `native_types()` error/crash/explode in a local build instead of returning data as they should.